_config.yml

```diff
@@ -19,3 +19,4 @@ exclude:
     - README.md
     - Rakefile
     - verify.rb
+    - vendor
```

Comment:
It looks like this could be a remnant of a very old version of twofactorauth.org, from back when we used the `vendor` directory. Is this intentional?